### PR TITLE
Don't show generated projects on Copr homepage

### PR DESCRIPTION
**Pull request by kontura**
Since we generate new projects quite often (for each PR) we shouldn't
spam the Copr homepage with them.

I think we can hard-code this but if an additional option for rgo would be preferred I can add it.

### CODE REVIEW

File: rgo/builders/copr.py [hunk 1]
```diff
@@ -70,7 +70,8 @@ def __init__(
                 name,
                 chroots=list(set(list(self.project.chroot_repos.keys()) + chroots)),
                 delete_after_days=delete_after_days,
-                additional_repos=additional_repos
+                additional_repos=additional_repos,
+                unlisted_on_hp=True
```
Comment (dmach) [inline rgo/builders/copr.py]: Could you revert this change?
I don't think we should be modifying any existing projects.

**Review by dmach — CHANGES_REQUESTED** [review]

File: rgo/builders/copr.py (hunk 1 above)
Comment (lukash) [reply rgo/builders/copr.py]: Agreed, that way existing projects like the nightlies will still show, which is IMO desired.

Comment (kontura) [reply rgo/builders/copr.py]: Sure.
